string name
string typeOfObstacle
string geometry
float32[] pose
float32[] dim